spec/services/pull_request/parse_repository_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe 'PullRequest::ParseRepository' do
+  context 'within given repo url' do
+    let(:repo_url) { 'https://github.com/example_company/example_project/pull/99' }
+
+    it 'should parse repo name and issue number' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.